### PR TITLE
fix: polymarket order filled filtering

### PR DESCRIPTION
**Pull request by md0x**
**Motivation**

There was a bug in the OrderFilled filtering in the polymarket notifier only considering buys of `outcomeToken0` and sells of `outcomeToken1`

**Summary**

Fix `OrderFilled` filtering

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #XXXX

### CODE REVIEW

File: packages/monitor-v2/src/monitor-polymarket/common.ts [hunk 1]
```diff
@@ -232,15 +232,19 @@ export const getOrderFilledEvents = async (
       const outcomeTokenOne = await Promise.all(
         events
           .filter((event) => {
-            return market.clobTokenIds[0] == event?.args?.takerAssetId.toString();
+            return [event?.args?.takerAssetId.toString(), event?.args?.makerAssetId.toString()].includes(
```
Comment (md0x) [inline packages/monitor-v2/src/monitor-polymarket/common.ts:235]: This filter was wrong, the way it worked before was filtering out sells and keeping buys.

File: packages/monitor-v2/src/monitor-polymarket/common.ts [hunk 2]
```diff
@@ -232,15 +232,19 @@ export const getOrderFilledEvents = async (
...
+            );
           })
           .map((event) => getTradeInfoFromOrderFilledEvent(params.provider, event))
       );
 
       const outcomeTokenTwo = await Promise.all(
         events
           .filter((event) => {
-            return market.clobTokenIds[1] == event?.args?.makerAssetId.toString();
+            return [event?.args?.takerAssetId.toString(), event?.args?.makerAssetId.toString()].includes(
```
Comment (md0x) [inline packages/monitor-v2/src/monitor-polymarket/common.ts:245]: We had similar issue here where only sells were considered before this fix.

**Review by Reinis-FRP — APPROVED** [review]
looking good

**Review by Reinis-FRP — APPROVED** [review]
looking good

**Pull request merged**